README.md

```diff
@@ -10,11 +10,15 @@ complexity of your layouts.
 A customisable Alert view is dynamically added to the Decor View of the Window, overlaying
 all content.
 
+[ ![Download](https://api.bintray.com/packages/tapadoo/maven/alerter/images/download.svg) ](https://bintray.com/tapadoo/maven/alerter/_latestVersion)
```

Comment:
Should move these badges to the top of the README. Are you able to inline them too?